src/main/java/com/entropyteam/entropay/common/BaseService.java

```diff
@@ -111,12 +112,19 @@ private Collection<Predicate> buildIdPredicates(Root<Entity> root, Filter filter
     }
 
     private Collection<Predicate> buildEntityPredicates(Root<Entity> root, Filter filter, CriteriaBuilder cb) {
-        if (MapUtils.isEmpty(filter.getGetByFieldsFilter())) {
+        if (MapUtils.isEmpty(filter.getGetByFieldsFilter()) && MapUtils.isEmpty(filter.getGetByDateFieldsFilter())) {
             return CollectionUtils.emptyCollection();
         }
         Collection<Predicate> predicates = filter.getGetByFieldsFilter().entrySet().stream().filter(f -> f.getKey() != SEARCH_TERM_KEY)
                 .map(f -> cb.equal(root.get(f.getKey()), f.getValue())).collect(Collectors.toSet());
 
+        if( filter.getGetByDateFieldsFilter().containsKey("dateTo") && filter.getGetByDateFieldsFilter().containsKey("dateFrom") ){
```

Comment:
please replace dateTo and dateFrom for the constant in ReactAdminMapper class. similar as we did for SEARCH_TERM_KEY